.changeset/serious-books-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
+---
+
+Changed e2e test case ids to remove duplicate case ids and fix inconsistencies between repo and Testmo. Thanks to this, our QA reports will be more accurate
```

Comment:
I don't see here any changes related with ids

Reply:
You're right - the changes were needed in Testmo, I've corrected the changeset description